editor/editor_node.cpp

```diff
@@ -1588,6 +1591,25 @@ void EditorNode::save_resource_as(const Ref<Resource> &p_resource, const String
 	file->popup_file_dialog();
 }
 
+void EditorNode::save_resource_bulk(const LocalVector<Ref<Resource>> &p_resource_bulk) {
```

Comment:
This is my attempt to reduce the save time. I need to check if it's still needed, though